cmd/generate/config/utils/generate.go

```diff
@@ -28,7 +28,7 @@ const (
 	// boundaries for the secret
 	secretPrefixUnique = `\b(`
 	secretPrefix       = `[\x60'"\s=]{0,5}(`
-	secretSuffix       = `)(?:[\x60'"\s;]|$)`
+	secretSuffix       = `)(?:[\x60'"\s;\\n]|$)`
```

Comment:
Slight mistake: this will match `\\` or `n` separately. I created #1719 to address this.